tests/demo.js

```diff
@@ -0,0 +1,17 @@
+// Interactive test of USNG methods
```

Comment:
I really like this idea and think we should allow consumers of the package to try it by adding it to the [bin](https://docs.npmjs.com/files/package.json#bin) section of the package.json. We should also update the readme to show people how they can try it out.